lua/dashboard/init.lua

```diff
@@ -192,6 +193,28 @@ function db:get_opts(callback)
   )
 end
 
+local function get_unique_buffer_name(opts)
```

Comment:
maybe it can receive buffer number then just do a format .

Reply:
This function actually does the format already, but not on a buffer number, but an integer from 1 to 9999, until it finds a unique buffer name. I don't think using buffer id is a good idea, it constantly increases, so we will soon end up with something like "Dashboard-77", which is confusing, IMO.

By default, the logic appends the integer at the end (like in the screenshot above), but if the name contains formatting sequences, like "%02d" for example, those will be used.